project/RELEASES.md

```diff
@@ -0,0 +1,168 @@
+    end
+    ocbReg -- no --> acc
+    debateReg -->|"decision on
+                   the recommended
+                   workarounds for
+                   the new regression"| acc
+    debateReg -->|"decision on
+                   the recommended
+                   workarounds for
+                   the old regression"| reject
+
+    regression -- no --> types
+    regIsLTS -- no --> types
+    types{"Can the fix
+           change types
+           in any correct
+           Scala 3 code?"}
+    -- yes --> request
+    types -- no --> ocbOther
+
+    request{"Is backport
+             of the fix
+             heavily requested?"}
+    -- yes --> debateReq
+    request -- no --> reject
+
+    debateReq[\"Possibility of
+                the backport is discussed
+                by the compiler team"/]
+    --> |"backport is rejected"| reject
+    debateReq --> |"backport is accepted"| ocbOther
+
+    subgraph "Other Fixes"
+        ocbOther{"Does it cause
+                 any new failures
+                 in the full CB?"}
+
+        -- yes -->
+        regFixOther[\"Try to provide
+                    a followup fix for
+                    a regressions"/]
+        -- success --> ocbOther
+
+        ocbOther -- no -->
+        lint{"Does it introduce
+              any new warnings
+              behind flags?"}
+        -- yes -->
+        lintOcb{"Does it cause any
+                 new failures in the full CB
+                 after forcing a new flag?"}
+        -- yes --> regFixOther
+    end
+
+    lint -- no --> acc
+    lintOcb -- no --> acc
+    regFixOther -- failure --> reject
+
+    acc(["The PR is backported"])
+    reject(["The PR is not backported"])
+```
+
+## How should things be backported?
+
+The backporting process is tracked by [a GitHub Project](https://github.com/orgs/lampepfl/projects/6) in the lampepfl organization. Every PR merged to the `main` branch is automatically added to the `Needs Assessment` column. Those PRs are reviewed by the release officer or other appointed person. They can decide to remove them from the project (backport rejected) or to move them to the `Backporting in progress` column. If the PR with the backport has any differences from the original PR, the person doing the backport will ask the author and reviewers of the original change to review the backport. After merging the backport, the PR will be moved to the `Backport done` column and, after the release, to the `Released` column.
+
+Maintainers can request backporting the entirety or a part of previously rejected PR by adding it to the `Backport requested` column.
+
+Labels from the `backport:*`  are used only for backports that targets versions with already released RCs.  They can be used to mark changes on the main that are fixing a critical bug present in the Scala Next RC release or changes that were backported to the future Scala LTS versions that should be also backported to the current RCs.
+
+## The release cycles
```

Comment:
At this point you will probably have lost many of the more casual followers of the blog, as the preceding section are flowchart were quite in-depth.

I would suggest also including the most important top level takeaways from this section, such as "Scala Next strictly follows a six-week release train model", in the intro to the blog post.